includes/Experiments.php

```diff
@@ -444,4 +438,22 @@ public function is_experiment_enabled( $name ) {
 		$experiments = get_option( Settings::SETTING_NAME_EXPERIMENTS );
 		return ! empty( $experiments[ $name ] );
 	}
+
+	/**
+	 * Returns the names of all enabled experiments.
+	 *
+	 * @since 1.4.0
+	 *
+	 * @return array List of all enabled experiments.
+	 */
+	public function get_enabled_experiments() {
+		$experiments = array_filter(
+			$this->get_experiments(),
+			function( $experiment ) {
+				return $this->is_experiment_enabled( $experiment['name'] );
+			}
+		);
+
+		return wp_list_pluck( $experiments, 'name' );
```

Comment:
```suggestion
		$experiments = array_filter(
			wp_list_pluck( $this->get_experiments(), 'name' ),
			[ $this, 'is_experiment_enabled' ]
		);
		
		return $experiments;
```

Is this a little cleaner?

Reply:
Nice!